### ISSUE TITLE
Comments on installing on a raspberry pi

### ISSUE THREAD

**Issue by kunoSchlonz** [paulknewton/fritz-monitor#6]
- it runs only in python3

- i need to install as well
`sudo apt-get install libatlas-base-dev`

- the requirements needed to run with --no-cache-dir

**Comment by paulknewton** [comment]
I have added python2 support with the future library.

On a raspberry pi, it is generally better to install the large libraries (scipy, numpy, pandas) etc via the apt package manager. This pulls in pre-built versions as well as any other libraries that are needed. It is also much, much quicker. I've added a note in the installation instructions to install these via "apt-get install..." first and only use "pip install -r requirements.txt" for the remaining libraries.

For --no-cache-dir, I am not sure why this is occurring on your platform. Perhaps there were some old versions previously installed, but this should normally cause a problem because "pip" download the required versions if needed. I think there isn't much I can propose on the code level.
